README.md

```diff
@@ -9,6 +9,7 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 ```
 git clone git@github.com:simonv3/beam.git
 cd beam
+yarn install
```

Comment:
```suggestion
yarn
```

These days we can actually just run `yarn`, it's equivalent to `yarn install`. Feel free to leave it the way it is if you think it's more clear as to what functionality is being performed that way though.